commit 74c68fa61b5ebf4c64605a3cc5e47154a66671ce
Author: Florian Weimer <fweimer@redhat.com>
Date:   Wed Mar 12 10:23:47 2025 +0100

    Linux: Remove attribute access from sched_getattr (bug 32781)
    
    The GCC attribute expects an element count, not bytes.

diff --git a/sysdeps/unix/sysv/linux/bits/sched.h b/sysdeps/unix/sysv/linux/bits/sched.h
index d4ac158d99e4c350..3bdd258bb987d49b 100644
--- a/sysdeps/unix/sysv/linux/bits/sched.h
+++ b/sysdeps/unix/sysv/linux/bits/sched.h
@@ -145,7 +145,7 @@ int sched_setattr (pid_t tid, struct sched_attr *attr, unsigned int flags)
    store it in *ATTR.  */
 int sched_getattr (pid_t tid, struct sched_attr *attr, unsigned int size,
 		   unsigned int flags)
-  __THROW __nonnull ((2)) __attr_access ((__write_only__, 2, 3));
+  __THROW __nonnull ((2));
 
 #endif
 
